css/index.css

```diff
@@ -48,99 +129,250 @@ table {
 }
 
 * {
-    box-sizing: border-box;
+	box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-    font-family: 'Roboto', sans-serif;
+html,
+body {
+	height: 100%;
+	font-family: 'Roboto', sans-serif;
 }
 
-h1, h2, h3, h4, h5 {
-    font-size: 18px;
-    margin-bottom: 15px;
-    font-family: 'Rubik', sans-serif;
+h1,
+h2,
+h3,
+h4,
+h5 {
+	font-size: 18px;
+	margin-bottom: 15px;
+	font-family: 'Rubik', sans-serif;
 }
 
 p {
-    line-height: 1.4;
+	line-height: 1.4;
 }
 
 .container {
-    width: 800px;
-    margin: 0 auto;
+	width: 800px;
+	margin: 0 auto;
+}
+
+.navigation {
+	display: flex;
+	width: 100%;
+	justify-content: space-around;
+	align-items: center;
+	margin: 5px;
+}
+
```

Comment:
Nice job with the navigation, consider the spacing on the design file, perhaps a bit more margin on top would help here.